### DISCUSSION TITLE
Vim syntax highlighting

### DISCUSSION THREAD

**Discussion by dlejay** [ghostty-org/ghostty#3494] [Q&A] [answered]
When launching Vim via `vim -u NONE` and then using `:syntax on`, I expect to have Vim syntax highlighting working. This is the case in Terminal.app or iTerm.
<img width="842" alt="Screenshot 2024-12-27 at 16 33 12" src="https://github.com/user-attachments/assets/0eea271b-4e51-41c4-bd51-85783d4386ca" />
But out of the box, this is not what I get with Ghostty.
<img width="912" alt="Screenshot 2024-12-27 at 16 29 59" src="https://github.com/user-attachments/assets/2516272f-2e6c-47f3-8f4f-c18b52f7702d" />
Ghostty:
* Version 1.0.0
* Build 8469

macOS 15.2

```
VIM - Vi IMproved 9.1 (2024 Jan 02, compiled Nov  9 2024 23:15:55)
macOS version - arm64
Included patches: 1-754
Compiled by root@apple.com
```

What have I missed ?

**Comment by dlejay** [comment, marked as answer]
Resolved: The issue was due to a non-virgin .zprofile where TERMINFO was redefined. Cleaning up the configuration resolved the issue

**Reply by ornago** [reply]
I deleted /home/user/.zprofile but it didn't helped. Still no syntax highlighting. Could you please explain a little more what you did?

**Reply by dlejay** [reply]
There are other places where TERMINFO could have been redefined. For example `zshrc`.
